app/views/works/_form.html.erb

```diff
@@ -0,0 +1,37 @@
+<%= form_with model: @work, class: 'create-work' do |f| %>
+<% if action_name == "Edit"%>
+  <h1> <%= action_name %> this <%= @work.category %> </h1>
+<% else %>
+  <h2> <%= action_name %> a new work </h2>
+<% end %>
+
+<section>
+  <div class="form-group">
+    <%= f.label :category %>
+    <%= f.select :category, ["album", "book", "movie"], {}, { class: "form-control" } %>
+  </div>
+  <div class="form-group">
+    <%= f.label :title %>
+    <%= f.text_field :title, { class: "form-control" } %>
+  </div>
+  <div class="form-group">
+    <%= f.label :creator %>
+    <%= f.text_field :creator, { class: "form-control" } %>
+  </div>
+  <div class="form-group">
+    <%= f.label :publication_year %>
+    <%= f.text_field :publication_year, { class: "form-control" } %>
+  </div>
+  <div class="form-group">
+    <%= f.label :description %>
+    <%= f.text_area :description, { class: "form-control" } %>
+  </div>
+  <% if action_name == "Edit"%>
+    <%= f.submit "Update Work", class: "btn btn-primary" %>
+  <% else %>
+    <%= f.submit "Create Work", class: "btn btn-primary" %>
+  <% end %>
```

Comment:
You could consider using another local variable here for "Update Work" and "Create Work" instead of a conditional.